File: 3st/Jaeyoung/옹알이 (1).cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<string> babbling) {
    int answer = 0;
    
    for(int i = 0; i < babbling.size(); i++) {
        int check = 1;
        for(int j = 0; j < babbling[i].size(); j++) {
            if(babbling[i].substr(j,3)=="aya") j += 2;
            else if(babbling[i].substr(j,2) == "ye") j += 1;
            else if(babbling[i].substr(j,3) == "woo") j += 2;
            else if(babbling[i].substr(j,2) == "ma") j += 1;
            else {
                check = 0;
                break;
            }
        }
        if(check == 1) answer++;
    }
    return answer;
}

File: 4st/Eunseong/옹알이 (1).java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        for (int i = 0; i < babbling.length; i++) {
            babbling[i] = babbling[i].replaceAll("aya|ye|woo|ma", "");
            if (!babbling[i].matches("[a-z]{1,}")) answer++;
            System.out.print(babbling[i] + " / ");
        }
        return answer;
    }
}